Libraries/Components/FocusableWindows/FocusableWindows.windows.js

```diff
@@ -86,7 +86,8 @@ class FocusableWindowsTemplate {
       'auto',
       'yes',
       'no',
-      'no-hide-descendants'
+      'no-hide-descendants',
+      'yes-dont-hide-descendants' // Windows only, not in ReactNative
```

Comment:
may be better to put it after 'yes'?

Reply:
Since this is not in RN I prefer to make merges from RN easier by putting all additional stuff at the end.

Reply:
Since this is not in RN I prefer to make merges from RN easier by putting all additional stuff at the end

---
In reply to: [184246079](https://github.com/Microsoft/react-native-windows/pull/1807#discussion_r184246079) [](ancestors = 184246079)